src/main/java/org/myrobotlab/framework/interfaces/ServiceInterface.java

```diff
@@ -134,4 +134,7 @@
 
   public String getFullName();
 
+  public void loadLocalizations();
+  public void loadLocalizations(String tag);

```

Comment:
"tag" should be called "language" or "locale"  it doesn't make sense to call it "tag"

Reply:
its been removed